images/bridge2/cmd/bridge/main.go

```diff
@@ -474,6 +492,22 @@ func (m *Main) Serve(ctx context.Context) {
 		sess := m.loadRequestSession(ctx, w, r)
 		m.serveSessionText(ctx, w, r, sess)
 	})
+	http.HandleFunc("/sessions/{sessID}/events", func(w http.ResponseWriter, r *http.Request) {
+		ctx, cancel := context.WithCancel(ctx)
+		defer cancel()
+		sess := m.loadRequestSession(ctx, w, r)
+		events := sessionEventStream(ctx, sess)
+		m.serveSessionUpdates(ctx, w, r, sess)
+
+		h := w.Header()
+		h.Set("Content-Type", "text/event-stream")
+		h.Set("Cache-Control", "no-store")
+
+		for e := range events {
+			fmt.Fprintf(w, "event: %s\nid: %s\n\n", e.Type, e.ID)
```

Comment:
So IIUC by setting the event type here we can use `addEventListener` on the client to listen for specific types. Though it appears `onmessage` is still called for everything if you want to listen to all events.

While the ID isn't necessary now, we could use it later to send data updates and handle reconnects resuming based on the last seen id.

Reply:
Oh, looking again it seems like `onmessage` is only called if there's not an `event:` type. Sending just the types does seem like it would make things easier to add a `data:` field later with the event data (instead of sending `data:` with some different content now).
For this to replace the `/data` websocket, we'd also want to send updates when a new session has been started to show in the sidebar. So, those events would probably need a distinct event type as well.